src/Sylius/Bundle/ResourceBundle/composer.json

```diff
@@ -37,9 +38,11 @@
     "require-dev": {
         "phpspec/phpspec": "^2.4",
         "akeneo/phpspec-skip-example-extension": "~1.2",
-        "doctrine/mongodb-odm": "1.0.*@dev",
+        "doctrine/mongodb-odm": "^1.0",
```

Comment:
It will affect 5.5 and 5.6 unless we add `echo "extension = mongo.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini` before installing dependencies.